x-pack/plugins/security_solution/public/common/containers/dashboards/api.ts

```diff
@@ -23,6 +23,7 @@ export const getDashboardsByTagIds = (
   abortSignal?: AbortSignal
 ): Promise<Dashboard[] | null> =>
   http.post(INTERNAL_DASHBOARDS_URL, {
+    version: '1',
```

Comment:
Do we version each endpoint independently?

Should we define a constant like `INTERNAL_API_VERSION` or `INTERNAL_DASHBOARDS_API_VERSION` in the `common` folder so we can reuse here and in UI?